src/fluree/server/handlers/create.clj

```diff
@@ -29,15 +30,13 @@
           (watcher/deliver-watch watcher tx-id persist-resp))))))
 
 (defn create-ledger
-  [p consensus conn watcher parsed-txn]
-  (let [{ledger-id const/iri-ledger
-         txn       "@graph"
-         opts      :opts} parsed-txn
-        [tx-id txn*] (normalize-txn txn)
+  [p consensus conn watcher expanded-txn]
+  (let [ledger-id     (-> expanded-txn (get const/iri-ledger) (get 0) (get "@value"))
```

Comment:
Starting to feel like we could use a `get-expanded-value` (or similar) fn along the lines of:

```clojure
(defn get-expanded-value
  [expanded iri-key]
  (-> expanded (get iri-key) first (get "@value"))
```

Reply:
Since this is dealing with `jld-processor/expand` output and not `json-ld/expand` output, I thought it would be confusing to have two virtually identical functions that won't work if you pick the wrong one. I know the manual unthreading is verbose but it's very readable and nobody will get bit by using the wrong helper down the road. I can still try to abstract if you like, but in this case I didn't think the abstraction was worth the cost.

Reply:
Hmm... not sure I follow. I saw the exact same sequence of fns applied in several places and suggested turning that into a fn that gets called instead. But maybe I missed the places that were slightly different? Proper namespacing & docstring might be good if you think it might o/w look tempting to misuse. But yeah, I think it makes sense to turn into a named fn. I'll leave the final decision for this PR up to you, though.